### ISSUE TITLE
Consider adding support to delete lockup contract account

### ISSUE THREAD

**Issue by frol** [NearDeFi/ft-lockup-contract#19]
This might be useful when the account was created by mistake or it has already served its purpose.

We discussed adding a new function which will add FULL ACCESS key and mark the contract immutable, but there is a risk that malicious actor will deploy an intermediate contract, alter the state (delete the immutability marker), and deploy lockup contract back again now having FULL control over the account.

Probably it is not worth the trouble, but I still wanted to discuss it anyway.

cc @evgenykuzyakov @alexaunter

**Comment by alexaunter** [comment]
I agree that it would be useful to delete the contract after it had served it's purpose, I am totally opposed to tolerating any human error here. I. e. the contract should be impossible to delete / claim access to while there is a nonzero amount of tokens are stored  in lockups or funded draft groups (no matter the state: locked or unlocked). Which, as you have said, limits the conditions in which the deletion is possible, making it's worthiness questionable.

tl;dr I am opposed to allowing removing the contract.

**Comment by evgenykuzyakov** [comment]
Yep, it's a large risk to have full access to save on the storage.
